src/pagination/renderer/ScrollBulletRenderer.ts

```diff
@@ -95,6 +90,29 @@ class ScrollBulletRenderer extends Renderer {
     const prevClassRegex = new RegExp(`^${prevClassPrefix}`);
     const nextClassRegex = new RegExp(`^${nextClassPrefix}`);
 
+    if (renderActiveBullet) {
+      const prevBullet = bullets[prevIndex];
+      if (prevBullet) {
+        const newBullet = this._createBulletFromString(
+          renderBullet(bulletClass, prevIndex),
+          prevIndex
+        );
+        prevBullet.parentElement!.replaceChild(newBullet, prevBullet);
+        bullets.splice(prevIndex, 1, newBullet);
```

Comment:
It seems that you can use it as follows without removing or creating an index.
```
bullets[prevIndex] = nextBullet;
```

Reply:
That would be better in this case 👍